ci/terraform/oidc/authdev2.tfvars

```diff
@@ -40,7 +41,7 @@ enable_api_gateway_execution_request_tracing = true
 spot_enabled                                 = false
 
 lambda_max_concurrency = 0
-lambda_min_concurrency = 0
+lambda_min_concurrency = 1
```

Comment:
Do we want to change this or was this a temp thing?

Reply:
I suggest changing this because of the following 2 reasons:
- authdev2 is sometimes very slow when more people are using it, this would help that
- we have a habit of keeping the configuration files for authdev1 and authdev2 the same, and this is currently enabled in authdev1 so I suggest doing it in authdev2 as well.

Reply:
Removed it as it seems to create problems with the deploys.